src/callbacks/get_callbacks.jl

```diff
@@ -327,6 +327,18 @@ function get_callbacks(config, sim_info, atmos, params, Y, p)
         )
     end
 
+    if parsed_args["external_forcing"] == "ReanalysisTimeVarying" &&
+       parsed_args["config"] == "column"
```

Comment:
```suggestion
    if parsed_args["external_forcing"] == "ReanalysisTimeVarying" &&
       parsed_args["config"] == "column"
```

What happens if you have `external_forcing` = "ReanalysisTimeVarying"` but `"config" = "sphere"`?